spec/acceptance/suites/ipa_fact/ipa_fact_spec.rb

```diff
@@ -12,6 +12,11 @@
 
   servers = hosts_with_role(hosts, 'server')
   servers.each do |server|
+    # Skip EL6 if FIPS is enabled
+    # IPA server on EL6 doens't support EL6, only EL7.4+
+    # See https://www.freeipa.org/page/Releases/4.5.0#FIPS_140-2_Support
+    next if (ENV['BEAKER_fips'] && server.platform == 'el-6-x86_64')
```

Comment:
This should check the host fact itself. Different providers (`gce`, `amazon`, etc...) may change this string

Reply:
I'm working on updating this now